Tests/TestExistentialAny/FixtureExistentialAny.swift

```diff
@@ -0,0 +1,52 @@
+let existentialAny = """
+/// \(String.mockAnnotation)
+protocol ExistentialAny {
+    var foo: P { get }
+    var bar: any P { get }
+    var baz: any P & Q { get }
+    var qux: (any P) -> any P & Q { get }
```

Comment:
Is it possible to check if `primary associated type` is also working with this change?

```
var a: any Collection<P> { get }
```